### DISCUSSION TITLE
Admin console under this manage account buffering error.

### DISCUSSION THREAD

**Discussion by 1996bkvidya** [keycloak/keycloak#33886] [Archive]
### Manage account buffering error in admin console page

> I have created aws eks cluster and all the necessary infrastructure using terraform.
> I have written yml files for deployment for keycloak page, service file, ingress file and i am using ngnix ingress controller. pods were able to create for all deployment component, service component, ingress component, and even assined domain name and certificate number for https and deployed keycloak application in production mode
> I am using external database AWS RDS Mariadb using JDBC driver

> Keycloak page is opening, and admin console page is opening, but when i click on manage account the page is buffering and when click on developers tool under network tab i am getting 403 forbidden error. 

> here is the code

<img width="134" alt="service-yml" src="https://github.com/user-attachments/assets/c19d3699-d01b-492f-85c4-340f76e5ee91">
<img width="332" alt="deployment-yml" src="https://github.com/user-attachments/assets/fec5fa2a-282e-4fc2-960b-472cb43234ab">
<img width="333" alt="deployment-yml2" src="https://github.com/user-attachments/assets/f70480bc-afbe-4ecc-937f-279fe3d09d98">
<img width="218" alt="deployment-yml1" src="https://github.com/user-attachments/assets/a3663c5e-fffb-4458-83b3-ba7143d71214">

> Please can you guys help me slove this issue.

**Comment by codespearhead** [comment]
1. The `vidyabk1996/tapplent-keycloak-mariadb` image is not an official Keycloak OCI image. Consider using the official Keycloak OCI image [1] and its corresponding Kubernetes CRD quickstart [2][3][4], or close this issue and contact the vendor of that OCI image for support.
2. You're using deprecated enviroment variables [3][4].

[1] https://github.com/keycloak/keycloak-quickstarts/blob/da307c4f291ece5c4d35d97ace8b2f3d09563e34/kubernetes/keycloak.yaml#L34
[2] https://www.keycloak.org/getting-started/getting-started-kube
[3] https://github.com/keycloak/keycloak/issues/33847
[4] https://github.com/keycloak/keycloak/issues/33847#issuecomment-2408642712
[5] https://github.com/keycloak/keycloak/discussions/31339
[6] https://github.com/keycloak/keycloak/discussions/31339#discussioncomment-10930705
